### PR TITLE
Course repository links

### PR DESCRIPTION
**Pull request by 0xf8f8ff**
- all links leading to the course, student and group repositories will open in a new browser tab
- course repository links on the teacher page will no longer attempt to open a popup window

### CODE REVIEW

File: public/src/components/teacher/groupHelper.tsx [hunk 1]
```diff
@@ -81,5 +81,5 @@ export function slugify(str: string): string {
 }
 
 export function generateGroupRepoLink(groupname: string, courseURL: string): JSX.Element {
-    return <a href={courseURL + slugify(groupname)}>{ groupname }</a>;
+    return <a href={courseURL + slugify(groupname)} target="_blank">{ groupname }</a>;
```
Comment (meling) [inline public/src/components/teacher/groupHelper.tsx]: groupName

File: public/src/components/teacher/Results.tsx [hunk 2]
```diff
@@ -116,7 +116,7 @@ export class Results extends React.Component<IResultsProp, IResultsState> {
     }
 
     private generateUserRepoLink(name: string, username: string): JSX.Element {
-        return <a href={this.props.courseURL + username + "-labs"}>{ name }</a>;
+        return <a href={this.props.courseURL + username + "-labs"} target="_blank">{ name }</a>;
```
Comment (meling) [inline public/src/components/teacher/Results.tsx]: userName

Comment (meling) [inline public/src/components/teacher/Results.tsx]: maybe this should use function or some other way to determine the user repo link. For example, maybe someone prefer to not add `-labs`?

File: public/src/components/teacher/groupHelper.tsx (hunk 1 above)
Comment (0xf8f8ff) [reply public/src/components/teacher/groupHelper.tsx]: Done.

File: public/src/components/teacher/Results.tsx (hunk 2 above)
Comment (0xf8f8ff) [reply public/src/components/teacher/Results.tsx]: And done.

Comment (0xf8f8ff) [reply public/src/components/teacher/Results.tsx]: Will be fixed (constant fetched from the server) as a part of the PR with Student Page data fetching optimization.

**Pull request merged**